Documentation/Storage-Configuration/Block-Storage-RBD/block-storage.md

```diff
@@ -59,7 +59,11 @@ parameters:
     # RBD image format. Defaults to "2".
     imageFormat: "2"
 
-    # RBD image features. Available for imageFormat: "2". CSI RBD currently supports only `layering` feature.
+    # RBD image features, equivalent to OR'd bitfield value: 63
```

Comment:
Is there a need to know the numeric value?
```suggestion
    # RBD image features
```

Reply:
Traditional Ceph sets these as a numeric value, so I wanted to make the congruence clear to people used to thinking in those terms.

Reply:
Blank lines are kind of inconsistent throughout the project, but I've taken out this one.

Reply:
https://docs.ceph.com/en/latest/rbd/rbd-config-ref/#image-features

It's the sum of each feature's "internal value", which are powers of two.  Old-school bitfields, OR them together, which works out to addition.  Layering alone would be `1`.  This tends to confuse people.

I've amended in what I think is the way you've requested.

Reply:
Thanks, makes sense now :)